tests/unit/activation/test_nushell.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import absolute_import, unicode_literals
+
+from virtualenv.activation import NushellActivator
+from virtualenv.info import IS_WIN
+
+
+def test_nushell(activation_tester_class, activation_tester):
+    class Nushell(activation_tester_class):
+        def __init__(self, session):
+            cmd = "c:\\program files\\nu\\bin\\nu.exe" if IS_WIN else "nu"
```

Comment:
This might work in the CI but will not work locally as well :+1: Better to use `shutil.which` here and only fallback to this hard coded path if that returns None and we're on windows.